README.md

```diff
@@ -140,44 +141,57 @@ Attributes on nodes and edges:
                     (add-attr 4 :label "node 4")
                     (add-attr-to-nodes :parity "even" [2 4])
                     (add-attr-to-edges :label "edge from node 5" [[5 6] [5 7]])))
-    
-    ; Return attribute value on node 1 with key :label               
+
+    ; Return attribute value on node 1 with key :label
     (attr attr-graph 1 :label)
     => "node 1"
-    
+
     ; Return attribute value on node 2 with key :parity
     (attr attr-graph 2 :parity)
     => "even"
-    
+
     ; Getting an attribute that doesn't exist returns nil
     (attr attr-graph 3 :label)
     => nil
-    
+
     ; Return all attributes for node 4
     ; Two attributes found
     (attrs attr-graph 4)
     => {:parity "even", :label "node 4"}
-    
+
     ; Return attribute value for edge between nodes 5 and 6 with key :label
     (attr attr-graph 5 6 :label)
     => "edge from node 5"
-    
+
     ; Return all attributes for edge between nodes 5 and 7
     (attrs attr-graph 5 7)
     => {:label "edge from node 5"}
-    
+
     ; Getting an attribute that doesn't exist returns nil
     (attrs attr-graph 3 4)
     => nil
-    
+
     ; Remove the attribute of node 4 with key :label
     (def attr-graph (remove-attr attr-graph 4 :label))
-    
+
     ; Return all attributes for node 4
-    : One attribute found because the other has been removed
+    ; One attribute found because the other has been removed
     (attrs attr-graph 4)
     => {:parity "even"}
 
+Derived graphs:
+
+    ; Build a derived graph of using a node mapping
+    (nodes (mapped-by #(+ 10 %) g))
+    => #{11 12 13 14 15 16 17 18 19}
+
+    ; Subgraphs of g
+    (edges (nodes-filtered-by #{1 2 3 5} dg))
+    => ([1 2] [2 1] [2 3] [3 2])
+
+    (edges (subgraph-starting-from dg 1))
```

Comment:
The name `subgraph-starting-from` reads a bit odd here given that the first argument is a graph and not a start node. Let's discuss potential renaming where the function is define and then update this accordingly.